### DISCUSSION TITLE
Ultimaker Cura 4.10.0-BETA

### DISCUSSION THREAD

**Discussion by rburema** [Ultimaker/Cura#9999] [Releases]
4.10.0-BETA

**Native CAD import plugin**
Ultimaker Professional and Excellence subscribers can now directly import native CAD files into Ultimaker Cura.
Enable this feature by downloading the Native CAD import plugin from the Ultimaker marketplace. 

**Flow visualization**
In preview mode you can now visualize the flow. Contributed by @khani3s.

**Show loading plugins on startup**
When starting Cura you will now see which plugins Cura is loading.

**Add Z position parameter to FilamentChange**
With the FilamentChange script you can now control all 3 coordinates. 

**Allow FilamentChange script to use Marlin M600 configuration**
Contributed by @Sekisback.

**Double click on file in Digital Factory**
When double clicking on a file in the open project dialog in Digital Factory it will now open in Cura. 

**Bug Fixes**
- Fixed temperature exceptions for UM2+C material profiles.
- Fixed a bug where pause at height stops all extrusion if relative extrusion is used.
- Fixed authentication issues when logging into UM account. Contributed by @fieldOfView.
- Fixed the pause-at-height retract with Repetier-flavour.
- Fixed erasing z-coordinate in the move tool to the value of 0. 
- Fixed the limit range of layer view to only visible structures. 
- Fixed a bug where Cura crashes when scaling a model on Linux. 
- Fixed path minimum limit. 
- Fixed a bug when using right to left language numbers were overlaying on text in the print settings.
- Fixed edge case with disabling bridging.
- Fixed a bug where some names with Unicode characters crashes Cura when trying to authorize. 
- Fixed renaming groups. Contributed by @fieldOfView. 
- Fixed a bug when the seam was not placed in sharpest corner.
- Fixed the gantry height for S-line printers. 
- Fixed a bug where a model is partially below build plate if center selected model is used.

**Printer definitions, profiles and materials.**
- Add CC0.4 core and materials for S3 and S5.
- Updated Axi machine start gcodes, contributed by @Synsuiter.
- Volumic 3D printer definitions, contributed by @VOLUMIC.
- Anycubic Mega X and Anycubic Mega S, contributed by @NilsRo.
- Updated Deltacomb printer profiles, contributed by @kaleidoscopeit.
- eMotionTech Strateo3D materials and profiles, contributed by @ChronosTech.
- Sovol SV03, contributed by @balacij.
- Two Trees Bluer, Bluer Plus, Sapphire Pro and Sapphire Plus, contributed by @Lester3DGadgets.
- Update Skriware 2, contributed by @Lukkoz.
- Longer LK1, LK1 Pro, LK1 Plus, LK4, LK4 Pro, LK5, LK5 Plus and Cube 2, contributed by @lowkeyjoe. 
- Mingda D3, D4 and Rock3, contributed by @cataclism. 
- JGAurora A6, contributed by CrissR.

<hr /><em>This discussion was created from the release <a href='https://github.com/Ultimaker/Cura/releases/tag/4.10-beta'>Ultimaker Cura 4.10.0-BETA</a>.</em>

**Comment by dxgldotorg** [comment]
Is there any timeframe for the integration of Arachne support?

**Reply by Zogar89** [reply]
Me too

**Reply by rburema** [reply]
We've decided to take it a bit slower, to make sure there will be less bugs on launch and give our materials team a proper chance to go over the changes so the profiles will be better suited for the new way of doing things.

We're currently still aiming to for this year, but have not set a definitive date for 5.0

P.S. Also, please put comments about Arachne in the latest discussion thread about the Arachne beta release, to keep it all on topic.

**Comment by mikeydoom** [comment]
Why hasn't there been a Ender 3 V2 machine profile added yet?

**Reply by fieldOfView** [reply]
Because it has not been contributed. Ultimaker only adds and maintains profiles for Ultimaker printers. All other printer profiles are submitted by 3rd parties.

**Reply by dxgldotorg** [reply]
Doubtful my 3D printer would be included either; it is a custom build.

**Reply by mikeydoom** [reply]
Well how would I make a submission of the Ender 3 V2?

I know all the dimensions, offsets, etc.

**Reply by balacij** [reply]
@mikeydoom https://github.com/Ultimaker/Cura/wiki/Adding-new-machine-profiles-to-Cura
You could probably copy/paste the Ender 3 files and update them for the V2, then just PR it to this repo.

**Reply by pcartwright81** [reply]
https://github.com/Ultimaker/Cura/pull/10216
